sdk/healthdataaiservices/azure-health-deidentification/pom.xml

```diff
@@ -43,6 +43,7 @@ Code generated by Microsoft (R) TypeSpec Code Generator.
   </developers>
 
   <properties>
+    <spotless.skip>false</spotless.skip>
```

Comment:
Why remove this?

Reply:
This isn't needed anymore as Spotless is always running

Reply:
I see. I will remove it in codegen/emitter.